quickstart/src/index.js

```diff
@@ -1,223 +1,150 @@
 'use strict';
 
-var Video = require('twilio-video');
-
-var activeRoom;
-var previewTracks;
-var identity;
-var roomName;
-
-// Attach the Track to the DOM.
-function attachTrack(track, container) {
-  container.appendChild(track.attach());
-}
-
-// Attach array of Tracks to the DOM.
-function attachTracks(tracks, container) {
-  tracks.forEach(function(track) {
-    attachTrack(track, container);
-  });
-}
-
-// Detach given track from the DOM.
-function detachTrack(track) {
-  track.detach().forEach(function(element) {
-    element.remove();
-  });
-}
-
-// Appends remoteParticipant name to the DOM.
-function appendName(identity, container) {
-  const name = document.createElement('p');
-  name.id = `participantName-${identity}`;
-  name.className = 'instructions';
-  name.textContent = identity;
-  container.appendChild(name);
-}
-
-// Removes remoteParticipant container from the DOM.
-function removeName(participant) {
-  if (participant) {
-    let { identity } = participant;
-    const container = document.getElementById(
-      `participantContainer-${identity}`
-    );
-    container.parentNode.removeChild(container);
-  }
-}
+const { isMobile } = require('./browser');
+const joinRoom = require('./joinroom');
+const micLevel = require('./miclevel');
+const selectMedia = require('./selectmedia');
+const selectRoom = require('./selectroom');
+const showError = require('./showerror');
+
+const $modals = $('#modals');
+const $selectMicModal = $('#select-mic', $modals);
+const $selectCameraModal = $('#select-camera', $modals);
+const $showErrorModal = $('#show-error', $modals);
+const $joinRoomModal = $('#join-room', $modals);
+
+const connectOptions = isMobile ? {
+  // Available only in Small Group or Group Rooms only. Please set "Room Type"
+  // to "Group" or "Small Group" in your Twilio Console:
+  // https://www.twilio.com/console/video/configure
+  bandwidthProfile: {
+    video: {
+      dominantSpeakerPriority: 'high',
+      maxSubscriptionBitrate: 2500000,
+      mode: 'collaboration',
+      renderDimensions: {
+        high: { height: 720, width: 1280 },
+        standard: { height: 90, width: 160 }
+      }
+    }
+  },
+
+  // Available only in Small Group or Group Rooms only. Please set "Room Type"
+  // to "Group" or "Small Group" in your Twilio Console:
+  // https://www.twilio.com/console/video/configure
+  dominantSpeaker: true,
+
+  // Uncomment this line to enable verbose logging.
+  // logLevel: 'debug'
+
+  // Comment this line if you are playing music.
+  maxAudioBitrate: 16000,
+
+  // Comment this line if you are in a Peer-to-Peer Room.
+  preferredVideoCodecs: [{ codec: 'VP8', simulcast: true }],
+
+  // For mobile browsers, capture 360p video @ 24 fps.
+  video: { height: 720, frameRate: 24, width: 1280 }
```

Comment:
@timmydoza I wanted to try how 720p video works with Bandwidth Profile APIs. It looked pretty promising. Can you try it out and let me know what you think? Maybe 360p is ideal for Peer-to-Peer Rooms.

Reply:
Sounds good, but I was asking because of the comment "For mobile browsers, capture 360p video @ 24 fps."

Looks like the mobile and desktop connectOptions both use `video: { height: 360, frameRate: 24, width: 640 }`. Is that intentional?

Reply:
@timmydoza The intention is to capture 720p for both mobile and desktop.

Reply:
I see. Should the comment be updated to `// For mobile browsers, capture 720 video @ 24 fps.` then?